LondonBritishLibrary/orient/BLorient790.xml

```diff
@@ -30,42 +31,55 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         </altIdentifier>
                         <altIdentifier>
                             <idno>Wright 375</idno>
-                            
+
                         </altIdentifier>
-                        
+
                     </msIdentifier>
                     <msContents>
                         <summary/>
                         <msItem xml:id="ms_i1">
                             <locus from="2r"/>
-                            <title type="complete" ref="LIT5188MashafaMadbal#Introduction"/>
+                            <title type="complete" ref="LIT5188MashafaMadbal#PartI" xml:lang="gez">
```

Comment:
I would be in favour of trying to keep a somewhat uniform appearance of the content of `<title>` in manuscripts. A great variance - Geez titles, titles provided by catalogues, no content etc. - leads to a very chaotic appearance on the app. I don't want to open an issue about this, because I do think that it makes sense to keep all possibilities open and not to overregulate this, but in my opinion, this is something to be kept in mind. I tend not to add content here if there are edited work records from which the title can be retrieved.

Reply:
Thank you! I have left the title here because I think that from the title in the work description it might not be clear that the first part of the work is meant here. This work is also completely new for me, and Wright provides a big number of titles which are not always easily attributed to the whole work or to its parts.